src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/RequestedFlowMapper.java

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.share.mappers;
+
+import org.openkilda.messaging.command.flow.FlowRequest;
+import org.openkilda.model.Flow;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public abstract class RequestedFlowMapper {
+
+    public static final RequestedFlowMapper INSTANCE = Mappers.getMapper(RequestedFlowMapper.class);
+
+    /**
+     * Convert {@link Flow} to {@link FlowRequest}.
+     */
+    @Mapping(target = "flowId", source = "flowId")
```

Comment:
I think you cat remove it because name is the same

Reply:
Did you mean "your cat"?

Reply:
:)